.github/workflows/check.yml

```
@@ -0,0 +1,40 @@
+name: Check pull request
+
+on:
+  push:
+    braches: ["*"]
```

actionlint (1.7.9)

unexpected key "braches" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)